### ISSUE TITLE
Previewing directories like files

### ISSUE THREAD

**Issue by jryom** [gokcehan/lf#701]
I was trying to figure out a way to use `tree` to preview directories, but it seems that `lf` will only allow custom preview functionality for files (is this correct or did I miss something?).

It would be great to also have the option of using custom previewing for directories.

And thanks for making this great tool!

**Comment by rickalex21** [comment]
@gokcehan I'm trying to display help in the preview, the problem is when I'm on a directory I can't see the output. Currently using Yad dialog box as a work around. Tree would be nice as well. Not sure if there's a better approach?

```sh
cmd help %{{ 
    sed -n  -E -e "/^# $1$/,/^$/p" ~/.config/lf/lfrc| yad --text-info 
}}
map Ht help "Tags"
```

Preview
```sh
sed -n  -E -e "/^# $1$/,/^$/p" ~/.config/lf/lfrc
exit 1
```
Thanks

**Comment by joelim-work** [comment]
This was fixed in #842, this issue wasn't linked properly in the PR.
